L1Trigger/Phase2L1ParticleFlow/interface/TauNNId.h

```diff
@@ -0,0 +1,32 @@
+#ifndef L1TRIGGER_PHASE2L1PARTICLEFLOWS_TAUNNID_H
```

Comment:
didn't realize this initially: this is a header file for a plugin, which generally aren't supposed to have header files. it would be better to combine this with the cc file.

Reply:
This header file is used by 
https://github.com/cms-sw/cmssw/blob/0ddd6404d8bdd2615dbe22c577c5eab8c3023073/L1Trigger/Phase2L1ParticleFlow/interface/L1NNTauProducer.h#L14
so moving its code .cc file does not seem like a good idea. Are there other suggestions? If the viable alternative is to leave the code as is, let's do that.

Reply:
Sorry, I commented in the wrong place. This comment was intended for `L1Trigger/Phase2L1ParticleFlow/interface/L1NNTauProducer.h`

Reply:
ok. done.